ocs_ci/deployment/aws.py

```diff
@@ -488,7 +492,7 @@ def create_rhel_instance(self):
                         },
                     },
                 ],
-                ImageId=config.ENV_DATA["rhel_worker_ami"],
+                ImageId=rhel_worker_ami,
```

Comment:
@petr-balogh , I was wrongly thought that we are setting some env var for imageid, but this is correct. I guess it should work smoothly unless we are using any rhel7 specific admin commands post deployment of instance.